CMakeLists.txt

```diff
@@ -67,15 +67,26 @@ if (SWIG_FOUND)
   else()
     message (STATUS "Searching for Ruby - found version ${RUBY_VERSION}.")
   endif()
+endif()
 
-  ########################################
-  # Include python
-  find_package(PythonInterp 3 REQUIRED) # change to Python3 when Bionic is EOL
-  find_package(PythonLibs QUIET)
-  if (NOT PYTHONLIBS_FOUND)
-    message (STATUS "Searching for Python - not found.")
+########################################
+# Python bindings
+find_package(PythonInterp 3 REQUIRED) # change to Python3 when Bionic is EOL
```

Comment:
have you considered `include(IgnPython)`?

* https://github.com/ignitionrobotics/sdformat/blob/sdf12/CMakeLists.txt#L62
* https://github.com/ignitionrobotics/ign-cmake/blob/ign-cmake2/cmake/IgnPython.cmake

Reply:
I have not, I was just moving existing code. Let me give that a try

Reply:
ca39f94

Reply:
it needs to be `include(IgnPython)`, not `find_package(IgnPython)`

Reply:
https://github.com/ignitionrobotics/ign-math/pull/360/commits/20104aed9ed38eb1cfb8b3a8c7aea6d32364b149

Reply:
Thanks. It's still unhappy though

```
CMake Error at /usr/lib/cmake/pybind11/FindPythonLibsNew.cmake:96 (message):
  Python config failure:

  Traceback (most recent call last):

    File "<string>", line 1, in <module>

  ImportError: cannot import name 'sysconfig' from 'distutils'
  (/usr/lib/python3.8/distutils/__init__.py)

Call Stack (most recent call first):
  /usr/lib/cmake/pybind11/pybind11Tools.cmake:16 (find_package)
  /usr/lib/cmake/pybind11/pybind11Config.cmake:100 (include)
  CMakeLists.txt:83 (find_package)
```

Reply:
It's green now with the dependency that was missing, thanks to @j-rivero